### PR TITLE
bruno: add version 1.2.0

### LINKED ISSUE #11969: [Request]: Bruno
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

bruno

### Description

Opensource IDE For Exploring and Testing Api's (lightweight alternative to postman/insomnia)

### Homepage

https://www.usebruno.com/

### Download Link(s)

https://github.com/usebruno/bruno/releases/download/v0.16.5/bruno_0.16.5_x64_win.exe

### Some Indication of Popularity/Repute

- [1169 stars](https://github.com/usebruno/bruno/stargazers) 
- [61 forks](https://github.com/usebruno/bruno/forks)
- featured on [Console #177](https://console.substack.com/p/console-177)

### PR DESCRIPTION
**Pull request by Zottelchen**
This PR adds the bruno manifest.

Closes #11969

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/bruno.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            ]
+        }
+    },
+    "shortcuts": [
+        [
+            "Bruno.exe",
+            "Bruno"
+        ]
+    ],
+    "checkver": "github",
```
[the remark this answers is not in the document]
Comment (Zottelchen) [reply bucket/bruno.json]: 👍

**Review by jonz94 — APPROVED** [review]

**Comment by jonz94** [comment]
I think the test failed because the latest version of `bruno` is 0.21.0, not due to any issues with the code.

So, updating the version and the hash should make the test pass.

**Comment by helloanoop** [comment]
Hi, This is Anoop. Creator of Bruno.

Looking forward to have this merged!

**Comment by thesobercoder** [comment]
@helloanoop Hi Anoop, any idea when this will be merged. Eagerly waiting!!

File: bucket/bruno.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.25.0",
+    "homepage": "https://www.usebruno.com/",
+    "license": "MIT",
+    "description": "Opensource IDE For Exploring and Testing Api's (lightweight alternative to postman/insomnia)",
```
Comment (tech189) [inline bucket/bruno.json:5]:
```suggestion
    "description": "Open source IDE for exploring and testing APIs (lightweight alternative to Postman/Insomnia)",
```
Improvement to spelling in description

**Comment by tech189** [comment]
Making a note that Bruno stores config data in `AppData\Roaming\bruno` so nothing to persist.

@Zottelchen could you update this manifest to the latest version please?

**Comment by Zottelchen** [comment]
@tech189 Sure, done.

File: bucket/bruno.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    },
+    "shortcuts": [
+        [
+            "Bruno.exe",
+            "Bruno"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/usebruno/bruno"
```
Comment (tech189) [inline bucket/bruno.json:20]:
```suggestion
            "hash": "98941b455b71d7c8c8bfff5c448c0587960962d947a2e37665e411bffb95d4d0"
        }
    },
    "extract_dir": "win-unpacked",
    "shortcuts": [
        [
            "Bruno.exe",
            "Bruno"
        ]
    ],
    "checkver": {
        "github": "https://github.com/usebruno/bruno"
```


Almost there, could you move `extract_dir` out of architecture and put it before `shortcuts` as per https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets ? That way if there's a new architecture in the future, the same directory can be used to extract for both.

**Review by tech189 — CHANGES_REQUESTED** [review]

**Review by tech189 — APPROVED** [review]

**Comment by tech189** [comment]
Thank you for your contribution!

**Pull request merged**